styleguide-content/visuell-identitet/farger.md

```diff
@@ -1,140 +1,637 @@
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-grey-warm">
+            <div>Lys varm grå</div>
+            <div>#F6F6F3<br/>@ffe-grey-warm</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-sand">
+            <div>Sand</div>
+            <div>#F8F5EB<br/>@ffe-sand</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-green-mint">
+            <div>Lys grønn</div>
+            <div>#E1F4E3<br/>@ffe-green-mint</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-blue-pale">
+            <div>Lys blå</div>
+            <div>#DFF1F9<br/>@ffe-blue-pale</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-orange-salmon">
+            <div>Lys oransje</div>
+            <div>#F3BBAA<br/>@ffe-orange-salmon</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-red">
+            <div>Rød</div>
+            <div>#E60000<br/>@ffe-red</div>
+        </li>
+    </ul>
+    <h5 class="ffe-h5">Bakgrunnsfarger tabeller</h5>
+    <ul class="sb1ds-color-palette">
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-blue-sky">
+            <div>Himmelblå (Accordion)</div>
+            <div>#7FC6E8<br/>@ffe-blue-sky</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-blue-pale">
+            <div>Lys blå (Hover/markering)</div>
+            <div>#DFF1F9<br/>@ffe-blue-pale</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-blue-ice">
+            <div>Isblå (Hover)</div>
+            <div>#EFF8FC<br/>@ffe-blue-ice</div>
+        </li>
+    </ul>
+    <h4 class="ffe-h4">Nøytrale farger</h4>
+    <ul class="sb1ds-color-palette">
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-black">
+            <div>Sort</div>
+            <div>#262626<br/>@ffe-black</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-grey-charcoal">
+            <div>Koksgrå</div>
+            <div>#676767<br/>@ffe-grey-charcoal</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-grey">
+            <div>Grå</div>
+            <div>#ADADAD<br/>@ffe-grey</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-grey-silver">
+            <div>Sølvgrå</div>
+            <div>#CCCCCC<br/>@ffe-grey-silver</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-grey-cloud">
+            <div>Skygrå</div>
+            <div>#F4F4F4<br/>@ffe-grey-cloud</div>
+        </li>
+        <li class="sb1ds-color-palette__item sb1ds-color-palette__item--ffe-white">
+            <div>Hvit</div>
+            <div>#FFFFFF<br/>@ffe-white</div>
```

Comment:
Hvit is also listed under Bakgrunnsfarger GUI. Is that intentional? Seems like the only one that has two listings.